lib/assesments/assessment_2/fi-as2-hira/Screens/Home/widgets/fi_as2_hira_card.dart

```diff
@@ -0,0 +1,126 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_interns/assesments/assessment_2/fi-as2-hira/constants/fi_as2_hira_colors.dart';
+
+class H1Cardbox extends StatelessWidget {
+  const H1Cardbox({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 320,
+      height: 400,
+      decoration: BoxDecoration(
+        boxShadow: const [
+          BoxShadow(
+            color: Constants.lightgrey,
+            blurRadius: 4.0,
+          ),
+        ],
+        color: const Color(0xFFfcfcfc),
+        borderRadius: BorderRadius.circular(20.0),
+      ),
+      child: Column(
+        children: [
+          ListTile(
+            leading: Container(
+              height: 60,
+              width: 60,
+              decoration: BoxDecoration(
+                  color: Constants.pink,
+                  borderRadius: BorderRadius.circular(50)),
+              child: Padding(
```

Comment:
Container has its own `padding:` property, you don't need to wrap the `child` in a `Padding` widget.